src/handlers/site_upload/powerset_merge.py

```diff
@@ -44,7 +44,7 @@ def __init__(self, event):
         self.study = s3_key_array[1]
         self.data_package = s3_key_array[2].split("__")[1]
         self.site = s3_key_array[3]
-        self.version = s3_key_array[4]
+        self.version = s3_key_array[4][-3:]
```

Comment:
Can we break on a `split` call or something? Assuming the length feels brittle.

Reply:
yeah, same as above, will change.